### DISCUSSION TITLE
Installation problem: "Client sent an HTTP request to an HTTPS server."

### DISCUSSION THREAD

**Discussion by arararar** [nextcloud/all-in-one#808] [Questions] [answered]
Hello out there,

i need a little help with installing, please!
i followed the instructions so far (https://github.com/nextcloud/all-in-one#how-to-use-this), i am using a fresh debian bullseye amd64 installation.
when i call "https://<server>:8443 the certificate is imported correctly.
i use a dyndns for <server>, updated via my fritz-box, ports 80, 8443 and 443 are forwarded to the debian outer system.
80 --> 80
443 --> 443
8443 --> 8443
but i get stuck at "new aio instance" with error message:

`Domain does not point to this server or the reverse proxy is not configured correctly. See the mastercontainer logs for more details. ('sudo docker logs -f nextcloud-aio-mastercontainer')`

and in the server console log:

`[client 127.0.0.1:36976] The response of the connection attempt to "http://<server>:443" was: Client sent an HTTP request to an HTTPS server., referer: https://<server>:8443/containers`

i replaced /etc/hostname and /etc/hosts-localhost entry with the dyndns domain-name <server>.

what am i missing?!
is my port forwarding scheme wrong? should i use 443 --> 8443 instead e.g.?

**Comment by szaimen** [comment]
Do you have an reverse proxy in between?

What is the startup command that you used?

Is something rewriting all access to your domain to a https connection?

**Reply by arararar** [reply]
Hi, thx for helping!!!
I did not configure any proxy, i only forwarded the ports.

Startup cmd was this the one given to install:
`For x64 CPUs:
sudo docker run -it \
--name nextcloud-aio-mastercontainer \
--restart always \
-p 80:80 \
-p 8080:8080 \
-p 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest`

Your last hint - i dont understand your question. ?

**Comment by szaimen** [comment, marked as answer]
See https://github.com/nextcloud/all-in-one#how-to-skip-the-domain-validation
